icon_validator/rules/plugin_validators/name_validator.py

```diff
@@ -0,0 +1,32 @@
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+class NameValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_name(name, plugin_name=False):
+        if not isinstance(name, str):
+            raise ValidationException("Name must not be blank")
+        if name == "":
+            raise ValidationException("Name must not be blank")
```

Comment:
We can connect that cases together 😄 
```suggestion
        if not isinstance(name, str) or name == "":
            raise ValidationException("Name must not be blank")
```